app/assets/stylesheets/views/_organisations.scss

```diff
@@ -110,6 +110,17 @@
   border-top-style: solid;
 }
 
+.organisation__section-border-top {
+  border-top: 5px solid $black;
+  padding-top: $gutter-half;
+}
+
+.organisation__corporate-information .organisation__float-section {
```

Comment:
Could this be more specific, like `.organisation__float-right`? Also does it need to nest below the corporate information class like this? Seems unnecessarily specific.

Frustrating that there isn't just a utility class to float right.

Reply:
I had to make it more specific so it overrode the `.grid-row .column-one-third` styling which floats it left (which makes it appear under the content when there's no FOI/Contact Info/High Profile Groups, rather than sticking to the right-hand side).